### PR TITLE
WordAds: Fix alignment issues

### LINKED ISSUE #30623: Gutenpack: Ads block visual layout
The different layouts don't display well, some make the block unusable for example mobile leaderboard:

screenshot 2019-02-06 at 17 51 20

### LINKED ISSUE #30625: Gutenpack: Ads block options
The Ads block has "Wide width" and "Full width" options, which I don't think make sense for this block:

screenshot 2019-02-06 at 17 53 26

We should just remove them. In fact I wonder if any of these options make sense.

### PR DESCRIPTION
**Pull request by enejb**
#### Changes proposed in this Pull Request
This PR intents to fix alignment issues found in the current WordAds block. 

The new block looks like this.
screen shot 2019-02-11 at 5 02 08 pm

#### Testing instructions
TBD...

Fixes #30623 
Fixes #30625

### CODE REVIEW

File: client/gutenberg/extensions/wordads/editor.scss [hunk 1]
```diff
@@ -1,6 +1,13 @@
-.aligncenter .wp-block-jetpack-wordads {
-justify-content:center;
+.wp-block-jetpack-wordads {
+	background: #FFF;
```
Comment (simison) [inline client/gutenberg/extensions/wordads/editor.scss]: We should prolly use `$white` sass variable here, or at least lowercase `#fff` to please `stylelint`.

File: client/gutenberg/extensions/wordads/editor.scss [hunk 2]
```diff
@@ -18,8 +25,19 @@ justify-content:center;
...
 	overflow: hidden;
 	flex-direction: column;
+	max-width: 100%;
 
 	.components-placeholder {
 		flex-grow: 2;
 	}
 }
+.jetpack-wordads-leaderboard .components-placeholder {
+	min-height:90px;
```
Comment (simison) [inline client/gutenberg/extensions/wordads/editor.scss]: `sylelint` warns about spacing

File: client/gutenberg/extensions/wordads/editor.scss [hunk 3]
```diff
@@ -1,6 +1,13 @@
-.aligncenter .wp-block-jetpack-wordads {
-justify-content:center;
+.wp-block-jetpack-wordads {
+	background: #FFF;
 }
+
+.wp-block-jetpack-wordads.aligncenter {
+	.jetpack-wordads__ad {
+			margin: 0 auto;
```
Comment (simison) [inline client/gutenberg/extensions/wordads/editor.scss]: extra tab :-)

**Comment by scruffian** [comment]
I tested this. LGTM

**Comment by dbspringer** [comment]
Units look good, ad unit sizes pulldown fits nicely into box too 👍

**Pull request merged**